src/documents/index.py

```diff
@@ -90,6 +92,7 @@ def open_index_searcher():
 def update_document(writer, doc):
     tags = ",".join([t.name for t in doc.tags.all()])
     tags_ids = ",".join([str(t.id) for t in doc.tags.all()])
+    comments = ",".join([str(c.comment) for c in Comment.objects.filter(document=doc)])
```

Comment:
I haven't looked at the Whoosh docs yet, but since a comment might contain commas, could there be a better separator to use?

Reply:
I don’t find the whoosh docs all that approachable but I’m not sure if it matters here? 

This is just constructing a string to use for searching but at the moment it’s not treating comments as individual objects but basically all of the comment text as another field on the doc.

Again not 100% certain

Reply:
Yeah, they're not the easiest to read.  It can always be changed if needed.